src/styled.js

```diff
@@ -27,18 +33,25 @@ const styled = ({tagName, elementStyle, mountSheet}: StyledArgs) => {
   return class StyledElement extends Component {
     static tagName: string = tagName
     static style: ComponentStyleType = elementStyle
+    static contextTypes = {
+      [channel]: PropTypes.object
+    }
```

Comment:
please do smth like this instead `static contextTypes = themeListener.contextTypes;`